### ISSUE TITLE
Server stuck monitoring old solution (re-open)

### ISSUE THREAD

**Issue by lhughey** [OYIon/LiveSharp#55]
The server is stuck monitoring another solution that has an old nuget. I've tried removing the server and reinstalling, but the server still starts up monitoring that old solution. 

I've tried the "reload" and the "force reload" method from the server menu, but it still wants to monitor that solution. **Can I server delete watch folder reference at start? Or can I manually delete a cache?**

This issue is identical to my previous ticket. https://github.com/OYIon/LiveSharp/issues/50
It is likely I can remove the error by opening up the project with the server is stuck on and updating it's outdated LiveSharp reference, but I shouldnt' need to.

**The details I see under the Application section on th**e server is:
 "Out of date Nuget package detected. (Installed 1.1.22 availabl 1.1.31) Please run update package Livesharp". 

**The log says:**
08:23:33 watcher created for at C:\Source\ProjectName\src\ProjectName\ 

(This project is not loaded in Visual studio and has not been loaded in 2 days, spanning multiple reboots of my computer).

**Comment by ionoy** [comment]
So, when you close the LiveSharp Server window and start it again (without starting the Xamarin app), you still see the error message?

**Comment by lhughey** [comment]
That is correct. I also see that error message after uninstalling and reinstalling the server.

**Comment by ionoy** [comment]
Is it possible, that the server is still running in the background? It should be named `Electron` if I'm not mistaken.

**Comment by lhughey** [comment]
Contacting you on gitter.
